### PR TITLE
replace toast with snackbar when publishing a post while offline

### LINKED ISSUE #9565: Vague error message when you create a new post while offline.
When you create a post while offline and publish it, you get a device is offline notice with a vague error message.

| |  |
|--------|-------|
||     |

## Tasks

- [ ] Replace with a Snackbar
- [ ] The message can be more clear. We should probably mention that it was not published and was saved as a draft instead.

### PR DESCRIPTION
**Pull request by adsonafonso**
Fixes #9565 

- Replaces toast with snackbar
- snackbar message - "Device is offline. Post saved as a draft."

Before: 
before

After:
offlinepublishing

To test:

1. Go to Blog Posts
2. Put the device on "Airplane Mode"
3. Create a new post while on Airplane mode and try to publish it.
4. A snackbar should appear with the "Device is offline. Post saved as a draft"

Other possible error message suggestions that i thought about:
-Unable to connect to internet. Post saved to drafts.
-A network error occurred. Post saved to drafts.
-Unable to publish due to a network error. Post saved to drafts

### CODE REVIEW

File: WordPress/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1481,7 +1481,7 @@
     <string name="error_publish_empty_post">Can\'t publish an empty post</string>
     <string name="error_publish_empty_page">Can\'t publish an empty page</string>
     <string name="error_save_empty_draft">Can\'t save an empty draft</string>
-    <string name="error_publish_no_network">Device is offline. Changes saved locally.</string>
+    <string name="error_publish_no_network">Device is offline. Post saved as a draft.</string>
```
Comment (maxme) [inline WordPress/src/main/res/values/strings.xml]: I think we should not use  "draft" or "local draft" here.

**Comment by adsonafonso** [comment]
updated screen with Snackbar on top of navigation
snackbar

File: WordPress/src/main/res/values/strings.xml (hunk 1 above)
Comment (adsonafonso) [reply WordPress/src/main/res/values/strings.xml]: updated the string to "Device is offline. Post Saved locally"

**Comment by maxme** [comment]
LGTM, :shipit:

**Pull request merged**